### PR TITLE
Update Video links on Firebase page

### LINKED ISSUE #9953: Fix YouTube links on Firebase page
### Page URL

https://docs.flutter.dev/data-and-backend/firebase

### Page source

[link](https://github.com/flutter/website/blob/main/src/data-and-backend/firebase.md)

### Describe the problem

Links for videos doesn't work

[Get to know Firebase for Flutter](https://github.com/flutter/website/blob/main/src/data-and-backend/%7B%7Byt-watch%7D%7D?v=4wunbF29Kkg) video workshop based on the codelab

[Using Firestore as a backend to your Flutter app](https://github.com/flutter/website/blob/main/src/data-and-backend/%7B%7Byt-watch%7D%7D?v=DqJ_KjFzL9I&t=38s) (video)
[Live Coding Firebase Authentication with Flutter](https://github.com/flutter/website/blob/main/src/data-and-backend/%7B%7Byt-watch%7D%7D?v=OlcYP6UXlm8) (video)
[Flutter & Firebase Auth 01](https://github.com/flutter/website/blob/main/src/data-and-backend/%7B%7Byt-watch%7D%7D?v=u_Lyx8KJWpg) (video)
[Flutter: Firebase Tutorial Part 1 - Auth and Sign in](https://github.com/flutter/website/blob/main/src/data-and-backend/%7B%7Byt-watch%7D%7D?v=13-jNF984C0) (video)

### Expected fix

fix the links so the can show the YouTube videos

### Additional context

_No response_

### I would like to fix this problem.

- [ ] I will try and fix this problem on docs.flutter.dev.

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #9953

### CODE REVIEW

File: src/data-and-backend/firebase.md [hunk 1]
```diff
@@ -3,28 +3,49 @@ title: Firebase
 description: How to use Firebase and Firestore with Flutter.
 ---
 
+{% include docs/yt_shims.liquid %}
+
+== Introduction
```
Comment (parlough) [inline src/data-and-backend/firebase.md:8]: Were you intending for these to be headers or some other new format?

If headers, please update to hash symbols for them to work as Markdown headers.

**Review by parlough — COMMENTED** [review]
Thanks for fixing the links, but question about the headers:

File: src/data-and-backend/firebase.md (hunk 1 above)
Comment (sfshaza2) [reply src/data-and-backend/firebase.md:8]: Yes, this is the first I've seen of new headers. We need to discuss this.

Comment (atsansone) [reply src/data-and-backend/firebase.md:8]: I mixed up MediaWiki format with Markdown. I'll fix it.

File: src/data-and-backend/firebase.md [hunk 2]
```diff
@@ -3,36 +3,57 @@ title: Firebase
...
 [chat app]: {{site.medium}}/flutter-community/building-a-chat-app-with-flutter-and-firebase-from-scratch-9eaa7f41782e
 [codelab1]: {{site.codelabs}}/codelabs/firebase-get-to-know-flutter
 [Firebase plugins]: {{site.firebase}}/docs/flutter/setup#available-plugins
 [started]: {{site.firebase}}/docs/flutter/setup
-[video]: {{yt-watch}}?v=DqJ_KjFzL9I&t=38s
-[video2]: {{yt-watch}}?v=OlcYP6UXlm8
-[video3]: {{yt-watch}}?v=u_Lyx8KJWpg
-[video4]: {{yt-watch}}?v=13-jNF984C0
-[workshop]: {{yt-watch}}?v=4wunbF29Kkg
+[video]: {{yt-watch}}?v#DqJ_KjFzL9I&t#38s
```
Comment (domesticmouse) [inline src/data-and-backend/firebase.md:55]: I don't think this is correct. Please revert.

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**